### PR TITLE
Adding news/jobs scheduling

### PR DESCRIPTION
**Pull request by houssem-yahiaoui**
As mentioned in issue #23 and #10 adding both regular posting for news over news channels and positing of job each Sunday, Tuesday, Thursday.

### CODE REVIEW

File: src/controllers/scheduler.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      .then(data => {
+        for (let i = 0, len = 5; i < len; i++) {
+          bot.say({
+            'attachments': [{
+              'fallback': 'Dyalna Jobs',
+              'color': '#36a64f',
+              'author_name': 'Dyalna Jobs',
+              'author_link': 'dyalna.com',
+              'title': `${ data.body[i].title }`,
+              'title_link': `${ data.body[i].url }`,
```
Comment (youknowriad) [inline src/controllers/scheduler.js]: could we use the dyalna jobs link :P

**Review by youknowriad — COMMENTED** [review]
I guess the code is duplicated between the crons and the controllers, could we factorize these handlers?

**Comment by houssem-yahiaoui** [comment]
@youknowriad, The handler are a bit different, since the showcasing mechanism aren't the same, yes same info source but different showcasing and different functions, so we can not.

File: src/controllers/scheduler.js (hunk 1 above)
Comment (houssem-yahiaoui) [reply src/controllers/scheduler.js]: @youknowriad didn't get it ? i'm using already the dyalna jobs links

Comment (youknowriad) [reply src/controllers/scheduler.js]: Something like `http://jobs.dyalna.com/job/${ slug }`, thoughts?

File: src/controllers/scheduler.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+const jobs = (bot) => {
+  cron.schedule('0 0 9 1-31 1-12 1,3,5', () => {
+    bot.say({
+      text : 'Good morning @here her\'s you jobs position',
+      channel: 'C3NRY5JP2',
+    })
+    superagent
+      .get('http://jobs.dyalna.com/jobs/api.php/job?category=1&page=1')
+      .then(data => {
+        for (let i = 0, len = 5; i < len; i++) {
```
Comment (youknowriad) [inline src/controllers/scheduler.js]: I personally prefer forEach, more functional :)

**Review by youknowriad — APPROVED** [review]
Awesome thanks

**Comment by houssem-yahiaoui** [comment]
haha i think you are happy now @youknowriad , you got back your forEach xD

**Pull request merged**